lib/Cake/View/Helper/FormHelper.php

```diff
@@ -2310,6 +2315,11 @@ public function month($fieldName, $attributes = array()) {
  * @link http://book.cakephp.org/2.0/en/core-libraries/helpers/form.html#FormHelper::hour
  */
 	public function hour($fieldName, $format24Hours = false, $attributes = array()) {
+		if (is_array($format24Hours)) {
+			$attributes = $format24Hours;
+			$format24Hours = null;
```

Comment:
Corrected the null to false: https://github.com/cakephp/cakephp/commit/c31fcd6f3993ad12d366567e03c357358bde40e3